File: website/docs/code-samples/screenshots/handler.java
View.OnClickListener oclBtnOk = new View.OnClickListener() {
    @Override
    public void onClick(View v) {
        Crowdin.sendScreenshot(YourActivity.this, screenshotName, new ScreenshotCallback() {
            @Override
            public void onSuccess() {
                Log.d("", "Screenshot uploaded");
            }

            @Override
            public void onFailure(Throwable throwable) {
                Log.d("", String.valueOf(throwable));
            }
      });
  }
};

File: website/docs/code-samples/screenshots/handler.kt
Crowdin.sendScreenshot(activity!!, screenshotName, object : ScreenshotCallback {
    override fun onSuccess() {
        Log.d(TAG, "Screenshot uploaded")
    }

    override fun onFailure(throwable: Throwable) {
        Log.d(TAG, throwable.localizedMessage)
    }
})